modules/nf-core/propr/propd/main.nf

```diff
@@ -10,17 +10,21 @@ process PROPR_PROPD {
     input:
     tuple val(meta), path(count)
     tuple val(meta2), path(samplesheet)
+    val GSEA
 
     output:
     tuple val(meta), path("*.propd.rds"), emit: propd
     tuple val(meta), path("*.propd.tsv"), emit: results
     tuple val(meta), path("*.fdr.tsv")  , emit: fdr         , optional:true
+    tuple val(meta), path("*.gct")  , emit: gct         , optional:true
+    tuple val(meta), path("*.cls")  , emit: cls         , optional:true
     path "*.R_sessionInfo.log"          , emit: session_info
     path "versions.yml"                 , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
+    GSEAopt = GSEA.toString().toUpperCase() // R expects upper case
```

Comment:
You can do this kind of small processings inside the template.
Or maybe you can modify the R code so that it will not expect upper case.

Reply:
This line has been removed from the code, since this parameter is now defined in the ext.args.